.gitignore

```diff
@@ -1,3 +1,5 @@
 dist/
+.stack-work/
 *.o
 *.hi
+TAGS
```

Comment:
git has a user-wide gitignore which is more appropriate for this kind of file than the project-specific gitignore.